### DISCUSSION TITLE
Is there a reason why div is the default element type for VisuallyHidden?

### DISCUSSION THREAD

**Discussion by BrendanBerkley** [adobe/react-spectrum#7144] [Q&A]
I just ran my markup through the HTML validator and I got a bunch of errors saying: `Element div not allowed as child of element button in this context.`

Buttons only want phrasing content and divs are flow content. I can change elementType to "span"; I get all of that.

But is there a reason why div is the default for this component? The thing is visually hidden, so the intrinsic things that div vs span bring shouldn't be relevant there. Are there any accessibility considerations here? Would it be a breaking change to change the default?

**Comment by snowystinger** [comment]
Ha! I never knew about this. Looks like they are trying to get it relaxed https://github.com/whatwg/html/issues/9705 as it seems a bit of a silly requirement.

I think it can be considered a false positive.

As for the default, we apply a margin on the element, and if I'm recalling correctly a `display: inline` element can't have a margin-top or margin-botom. There may be some other styles that wouldn't apply correctly but I don't remember them off the top of my head.

**Reply by BrendanBerkley** [reply]
Interesting discussion over there, thanks for finding it.
